gunpowder_augmentor.py

```diff
@@ -0,0 +1,106 @@
+import zarr
+import gunpowder as gp
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import torch
+
+
+def process_zarr_data(load_path, output_shape=(30, 128, 128), stack_size=5,  device='cuda'):
```

Comment:
Consider renaming function to `prepare_gunpowder_pipeline`

Reply:
renamed!